src/Umbraco.Web.Common/Extensions/LinkGeneratorExtensions.cs

```diff
@@ -135,5 +136,22 @@ public static string GetUmbracoApiService(this LinkGenerator linkGenerator, stri
             }
             return linkGenerator.GetUmbracoApiService(actionName, ControllerExtensions.GetControllerName(apiControllerType), area, id);
         }
+
+        public static string GetUmbracoApiService<T>(this LinkGenerator linkGenerator, Expression<Func<T, object>> methodSelector)
+            where T : UmbracoApiController
+        {
+            var method = ExpressionHelper.GetMethodInfo(methodSelector);
+            var methodParams = ExpressionHelper.GetMethodParams(methodSelector);
+            if (method == null)
+            {
+                throw new MissingMethodException("Could not find the method " + methodSelector + " on type " + typeof(T) + " or the result ");
```

Comment:
Expect this is migrated rather than added, but could use the more modern string interpolation here.

i.e. `throw new MissingMethodException($"Could not find the method {methodSelector} on type {typeof(T)} or the result.");`